src/engine/test/source/builder/builders/opBuilderHelperSHAfrom_test.cpp

```diff
@@ -0,0 +1,161 @@
+/* Copyright (C) 2015-2022, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <any>
+#include <gtest/gtest.h>
+#include <vector>
+
+#include <baseTypes.hpp>
+
+#include "opBuilderSHAfrom.hpp"
+
+using namespace base;
+namespace bld = builder::internals::builders;
```

Comment:
```suggestion
using builder::internals::builders;
```
And please remove every `bld::` occurrences.

Reply:
`namespace bld ...` is used in 34 tests right now. while using `builder::` just in 9.
Before changing we should define which will be using from now on and correct the ones that are different. From my perspective bld it's clearer.